content/doc/book/blueocean/creating-pipelines.adoc

```diff
@@ -77,11 +77,15 @@ to click the *Create Pipeline* button.
 Blue Ocean will then scan your local repository's branches for a `Jenkinsfile`
 and will commence a Pipeline run for each branch containing a `Jenkinsfile`. If
 Blue Ocean cannot find any `Jenkinsfile`, you will be prompted to begin creating
-one through the <<pipeline-editor#,Blue Ocean Pipeline editor>>.
+one through the <<pipeline-editor#,Pipeline editor>>.
 
 
 ==== Remote repository
 
+Since the Pipeline editor saves edited Pipelines to Git repositories as
+`Jenkinsfile`&zwj;s, Blue Ocean only supports connections to remote Git
+repositories over the SSH protocol.
```

Comment:
Ah - it took me a while to find out the reason for this change - I guess this is what happens when you start work on a branch (like this one) and leave it for a while 🤦‍♂️ .

The rationale for this was from a comment that @rtyler made on a previous PR of mine [here](https://github.com/jenkins-infra/jenkins.io/pull/1294#discussion_r158429837) (which only seems to open correctly in Firefox - not Chrome).

Essentially, I think it made more sense moving the core content of that admonition to the top of this section (i.e. about SSH protocol support) and remove the content about HTTP or HTTPS (because when you type URLs starting with `http...` or `https...` into the Blue Ocean UI, a big warning pops up anyway telling you that these protocols are not supported).